.github/ISSUE_TEMPLATE/bug-report---other.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report - Other
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
I think making a vcpkg and port bug report template with default labels is worthwhile.